Description: Don't break compiler security flags.
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 tv-grab-dvb (0.9+r90-1~cyber1) UNRELEASED; urgency=low
 .
   * New upstream release
Author: Trent W. Buck <twb@cyber.com.au>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- tv-grab-dvb-0.9+r90.orig/Makefile
+++ tv-grab-dvb-0.9+r90/Makefile
@@ -1,9 +1,5 @@
 #!/usr/bin/make -f
 
-#Perhaps you want a line like this instead. I've not used autoconf yet
-#CFLAGS=-Wall -O2 -I/usr/src/dvb-kernel/linux/include/
-CFLAGS=-Wall -O0 -g
-
 dvb_text := dvb_text.o
 dvb_text := dvb_text_iconv.o
 
